source/class/qx/tool/cli/commands/Package.js

```diff
@@ -131,7 +131,7 @@ qx.Class.define("qx.tool.cli.commands.Package", {
 
     /**
      * Convenience method to return all config file models as an array
-     * @return {Promise<[{qx.tool.config.Manifest}, {qx.tool.config.Lockfile}, {qx.tool.config.Compile}]>}
+     * @return {Promise}  [qx.tool.config.Manifest, qx.tool.config.Lockfile]
```

Comment:
This also seems to have corrupted the type hint

Reply:
in es6, the function should be `async`